src/models/comment.js

```diff
@@ -0,0 +1,22 @@
+//Requires Mongoose
+const mongoose = require("mongoose");
+
+//Creates variables to store Schema
+
+const Schema = mongoose.Schema;
+
+//Creates comment schema
+
+const commentSchema = Schema({
+  userID: { type: Schema.Types.ObjectId, ref: "User" },
+  username: { type: Schema.Types.ObjectId, ref: "User" },
+  text: String,
+  date: { type: Date, default: Date.now() },
+  channel: String
```

Comment:
Do we need to channel key right now?  We have only one channel so we are displaying all the comments.